projects/zeeves/project.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
zeeves seems to have formatting issues to the frontmatter. not displaying properly